src/test/java/codeu/controller/ActivityFeedServletTest.java

```diff
@@ -41,12 +52,32 @@ public void setup() {
         mockRequestDispatcher = Mockito.mock(RequestDispatcher.class);
         Mockito.when(mockRequest.getRequestDispatcher("/WEB-INF/view/activityFeed.jsp"))
                 .thenReturn(mockRequestDispatcher);
+
+        mockActivityStore = Mockito.mock(ActivityStore.class);
+        activityFeedServlet.setActivityStore(mockActivityStore);
     }
 
     @Test
-    public void testDoGet() throws IOException, ServletException {
+    public void testDoGet() throws IOException, ServletException, PersistentDataStoreException {
+        Activity activity = new Activity(UUID.randomUUID(), "User1 joined!", Instant.now(), ActivityTypeEnum.USER_ADDED);
+        List<Activity> activities = new ArrayList<>(Arrays.asList(activity));
+
+        Mockito.when(mockActivityStore.getActivitiesBeforeDatetime(Mockito.any(Instant.class), Mockito.anyInt()))
+                .thenReturn(activities);
+
+        activityFeedServlet.doGet(mockRequest, mockResponse);
+        Mockito.verify(mockRequest).setAttribute("activities", activities);
+        Mockito.verify(mockRequest, Mockito.never()).setAttribute("error", Mockito.eq(Mockito.notNull())); //error is never set
```

Comment:
Ensure that under normal conditions we are not setting the `error` attribute on the request object